src/app/components/bitcoin-fees-list/use-bitcoin-fees-list.ts

```diff
@@ -75,36 +79,46 @@ export function useBitcoinFeesList({
       feeRate: feeRates.hourFee.toNumber(),
     };
 
+    const feesArr = [];
```

Comment:
Maybe you can spread values into this array instead?

```
    ...(highFeeValue ? [{
        label: BtcFeeType.High,
        value: highFeeValue,
        btcValue: formatMoneyPadded(createMoney(highFeeValue, 'BTC')),
        time: btcTxTimeMap.fastestFee,
        fiatValue: getFiatFeeValue(highFeeValue),
        feeRate: feeRates.fastestFee.toNumber(),
      }] : [])
  ]
```

Reply:
I made it so to improve readability. don't really like such constructions even if they save several lines of code